po/README.md

```diff
@@ -0,0 +1,55 @@
+# Support for International Languages
```

Comment:
Reviewers: What is missing from this file that would be helpful for developers, collaborators, translators that wish to update Ledger's codebase and/or translations?